scipy/optimize/_minimize.py

```diff
@@ -307,7 +307,11 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
        Center -- Institute for Flight Mechanics, Koln, Germany.
     .. [13] Conn, A. R., Gould, N. I., and Toint, P. L.
        Trust region methods. 2000. Siam. pp. 169-200.
-
+    .. [14] Gao, F. and Han, L.
```

Comment:
If you add a ref here it's best to reference it in the text above

Reply:
I guess the reference should be in `_minimize_neldermead()` below where the citation is added.